docs/guides/es2k/compiling-p4-programs.md

```diff
@@ -1,135 +1,133 @@
 # Compiling P4 Programs for ES2K
 
-## 1. Overview
+This document explains how to install and use the Intel&reg; IPU E2100 P4Tools
+Container to compile a P4 program to build artifacts, including the `.pkg`
+file, that can be deployed on ES2K.
 
-This document explains how to install the Intel&reg; IPU E2100 P4 Compiler
-(`p4c-pna-xxp`) and Configurable Pipeline Tool (`cpt`), and use
-them to compile a P4 program
+The list of supported host OSes are listed in the "Supported Operating Systems"
+section of the IPU Software User Guide (RDC Doc#778226), henceforth referred to
+as the "IPU SWG".
 
-## 2. Installing the P4 Compiler and Tools
+## Build and run the P4 Tools container
 
-### 2.1 Install RPMs
+The tools required to compile and build a custom p4 package
+can be installed by building and running the P4 Tools Container.
```

Comment:
```suggestion
can be installed by building the P4 Tools Container.
```
You _install_ the tools by building the container.
You _use_ the tools by running the container.

Reply:
Done